### ISSUE TITLE
GPS location - range

### ISSUE THREAD

**Issue by vukisoft** [stevenblair/commuter-bliss-uk#7]
Hey again :)
Did you change the distance range for the GPS location with last upgrade? It used to work from my home and after the upgrade it doesn't. Not even if I switch "will walk" option.
Would it be possible to reset the range what it used to be? Or add an option to configure it?

**Comment by stevenblair** [comment]
Yes, there is a check for whether or not it is feasible to walk to a station, but this is fairly inaccurate (especially for large stations with multiple entrances) and had a hard-coded maximum. This maximum walking time is now configurable in the Settings page - try increasing this value and let me know if it doesn't help in your case.

**Comment by vukisoft** [comment]
What was the original hard-coded value (that worked from my home over the last 1-2 years :))?

**Comment by vukisoft** [comment]
Thx, works now!

However results are still unreliable, today it did not show the next train but from the one after that. Later on it showed the next, but it appeared at the bottom line, not the top .. Any idea about these issues? Never occurred before the last "major" upgrade.
